### PR TITLE
docs: how to use the addons in an Angular component

### PR DESCRIPTION
**Pull request by profanis**
## What I did

I worked a bit with the addons and I liked that feature a lot. While the code snippet for this part was missing in the Angular documentation I decided to add it. 

## How to test

1. Follow the instructions of this [doc](https://storybook.js.org/docs/angular/addons/writing-addons) and create a panel addon. Make sure to follow all the steps up to "Using the addon with a story" (without this).
2. Create a root level `preset.js`. [(details here)](https://storybook.js.org/docs/angular/addons/writing-addons#root-level-presetjs)
2. Create an Angular application and init a storybook
3. In the `.storybook/main.js` add in the `addons` array property the absolute path of the add addon of step 1

### CODE REVIEW

**Review by ThibaudAV — APPROVED** [review]

**Comment by jonniebigodes** [comment]
This is good on my end. @profanis and @ThibaudAV i'm planning on pushing the remainder Angular snippets sometime next week. I'm currently waiting on feedback on one other pull request regarding snippets and depending on how it goes i'll adjust accordingly and i'll push them through.

For now i'm good with it. Thanks once again for the time and effort you put into the pull request.

@shilman i'm going to self merge this if you don't mind.

**Pull request merged**
